Source/Data/Blackjack.Data/Repos/UserRepo.cs

```diff
@@ -74,6 +83,43 @@ public async Task<List<UserEntity>> GetUsersBySoundex(string soundex)
 
         return userEntities;
     }
+
+    public async Task<List<UserEntity>> UpdateUser(UserEntity user)
+    {
+        List<SqlParameter> parameters = new List<SqlParameter>()
+            {
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.UserID)}",
+                    SqlDbType = System.Data.SqlDbType.Int,
+                    Value = user.UserID
+                },
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.Username)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.Username
+                },
+/*                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.UserSoundex)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.UserSoundex
+                },*/
+                new SqlParameter()
+                {
+                    ParameterName = $"@{nameof(user.Balance)}",
+                    SqlDbType = System.Data.SqlDbType.VarChar,
+                    Value = user.Balance
+                }
+            };
+
+        DataTable queryData =
+            await base.Update(StoredProcedures.UpdateUser, parameters);
+
+        List<UserEntity> userEntities = CreateUserEntity(queryData);
+        return userEntities;
```

Comment:
This method shouldn't return anything

Reply:
Makes sense to me, but I just copied the format from my practice API. I'm just curious if that was just something we did for practice then or if there would have been a reason in that if you can recall. Also, I'm assuming this means all the other parts of the chain can just remove their return functions

Reply:
In practice if nothing is autogenerated by the database you won't need to do what we did in the PracticeAPI, but we didn't have a scenario of the db generating data on update so we used the pattern as though it did to see how it'd be done.

In this application when updating a user nothing is programmatically changed such as a ModifiedDatetime, or a unique ID or something, so you don't need to fetch the user from the Db, your code contains all the latest data already.